app/charts/chart-config-ui-options.ts

```diff
@@ -1265,6 +1269,13 @@ const chartConfigOptionsUISpec: ChartSpecs = {
           },
         },
       },
+      {
+        idAttributes: ["componentId"],
```

Comment:
@ptbrowne I think we can use an empty array here and also change `baseLayer` to not have this – as far as I am aware, we do not have a `componentId` property in base layer and custom layers, I think it was wrong before 😱